experimental/envoy_proxy/client.yaml

```diff
@@ -0,0 +1,47 @@
+static_resources:
```

Comment:
I'm not sure I follow the hierarchy here, why are client and server files mixed in the same dir? Isn't it cleaner to have separate dirs?

Reply:
Created separate dirs